src/django/sqls/remove_items_where_facility_id_is_null.sql

```diff
@@ -0,0 +1,56 @@
+CREATE OR REPLACE PROCEDURE remove_facilitylistitems_where_facility_id_is_null()
+LANGUAGE plpgsql
+AS $$
+BEGIN
+    -- Update facility_id in api_facilitylistitem
+    UPDATE api_facilitylistitem
+    SET facility_id = api_facility.id
+    FROM api_facility
+    WHERE api_facilitylistitem.id = api_facility.created_from_id
+    AND api_facilitylistitem.facility_id IS NULL;
+
+    -- Create temporary tables to store the IDs of the items that have facility_id as NULL
+    CREATE TEMPORARY TABLE temporary_facility_list_item_ids AS
+    SELECT id 
+    FROM api_facilitylistitem 
+    WHERE facility_id IS NULL;
+
+    CREATE TEMPORARY TABLE temporary_facility_list_item_temp_ids AS
```

Comment:
Why do we need temp table, could we just store it in variable to use?

Reply:
Changed using temporary tables to using variables.